nmi_tools/nmi-build-platforms

```diff
@@ -1,23 +1,23 @@
 # One platform per line. Comments (starting with #) and blank lines are ignored.
 
 # Officially supported platforms
-docker://htcondor/nmi-build:aarch64_AlmaLinux8-23102100
-docker://htcondor/nmi-build:aarch64_AlmaLinux9-23102100
-docker://htcondor/nmi-build:ppc64le_AlmaLinux8-23102100
-docker://htcondor/nmi-build:x86_64_AlmaLinux8-23102100
-docker://htcondor/nmi-build:x86_64_AlmaLinux9-23102100
-docker://htcondor/nmi-build:x86_64_AmazonLinux2023-23102100
+docker://htcondor/nmi-build:aarch64_AlmaLinux8-23102200
+docker://htcondor/nmi-build:aarch64_AlmaLinux9-23102200
+docker://htcondor/nmi-build:ppc64le_AlmaLinux8-23102200
+docker://htcondor/nmi-build:x86_64_AlmaLinux8-23102200
+docker://htcondor/nmi-build:x86_64_AlmaLinux9-23102200
+docker://htcondor/nmi-build:x86_64_AmazonLinux2023-23102200
 docker://htcondor/nmi-build:x86_64_CentOS7-23070200
-docker://htcondor/nmi-build:x86_64_Debian11-23102100
-docker://htcondor/nmi-build:x86_64_Debian12-23102100
-docker://htcondor/nmi-build:x86_64_openSUSE15-23102100
-docker://htcondor/nmi-build:x86_64_Ubuntu20-23102100
-docker://htcondor/nmi-build:x86_64_Ubuntu22-23102100
-docker://htcondor/nmi-build:x86_64_Ubuntu24-23102100
+docker://htcondor/nmi-build:x86_64_Debian11-23102201
+docker://htcondor/nmi-build:x86_64_Debian12-23102201
+docker://htcondor/nmi-build:x86_64_openSUSE15-23102200
+docker://htcondor/nmi-build:x86_64_Ubuntu20-23102201
+docker://htcondor/nmi-build:x86_64_Ubuntu22-23102201
+docker://htcondor/nmi-build:x86_64_Ubuntu24-23102201
```

Comment:
Just want to verify that these are supposed to end with `1` and not `0`

Reply:
The proper Pelican (7.14.1) was not in the Docker image for version 23102200 on Debian and Ubuntu systems. So, for Debian and Ubuntu, I needed to rebuild and got version 23102201.